cmd/ttn-lw-cli/commands/gateways.go

```diff
@@ -471,6 +472,44 @@ var (
 			return io.Write(os.Stdout, config.OutputFormat, res)
 		},
 	}
+	gatewaysBatchConnectionStats = &cobra.Command{
+		Use:     "batch-get-connection-stats",
+		Aliases: []string{"batch-connection-stats", "batch-cnx-stats"},
+		Short:   "Get connection stats for a batch of gateways",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ids, err := cmd.Flags().GetString("gateway-ids")
```

Comment:
You can use `GetStringSlice` (and register the parameter as a string slice) instead.

References https://github.com/spf13/pflag/blob/85dd5c8bc61cfa382fecd072378089d4e856579d/string_slice.go#L88-L95 .